setup.py

```diff
@@ -14,10 +14,15 @@ def find_version():
             raise RuntimeError("Unable to find version string.")
 
 
+with open('README.rst') as f:
```

Comment:
It's better to use `io.open('README.rst', encoding='utf-8')` to prevent the following problems and be compatible with Python 2 and 3.

https://github.com/RGF-team/rgf/issues/65
https://github.com/henry0312/pytest-codestyle/pull/34

Reply:
Thanks, updated!